### ISSUE TITLE
Developer Portal Landing Page

### ISSUE THREAD

**Issue by buckyroberts** [thenewboston-developers/Website#1901]
## Overview
We are going to be creating a new Developer Portal where all of our technical resources such as API documentation, information on creating projects, network architecture and guides, etc... are going to live. The overall feature will include multiple pages and different sections, but we will be working on them in separate steps to keep things simple.

## Task
For the first task, we are only going to be creating the landing page. A few notes about this feature:

- We can create this page but do not include a link in the top navigation yet. We will develop and test it by visiting the URL directly for now.
- Some of the links on the page are going to be to other sections/pages that are not developed yet. For now you can just link them to a temp URL like `https://www.google.com/` or `#`.

In short, we are basically just designing the UI for this "hidden" page right now. This way we can develop iteratively without causing a huge merge conflict. Later on we will include the link in the top navigation to make everything public.

Responsive breakpoints are included in the `UI - Landing page` Figma file. For the top background element, we will be using the Particles library here: https://vincentgarreau.com/particles.js/

https://www.figma.com/file/PRQm1ArLsBy1aZh2rZFwDz/Developer-Portal-s-Landing-Page?node-id=625%3A0
